config/gasSchedule.go

```diff
@@ -725,3 +723,10 @@ func FillGasMap_WASMOpcodeValues(value uint64) map[string]uint64 {
 
 	return gasMap
 }
+
+const GasValueForTests = 1
+const AsyncCallbackGasLockForTests = 100_000
+
+func MakeGasMapForTests() GasScheduleMap {
+	return MakeGasMap(GasValueForTests, AsyncCallbackGasLockForTests)
```

Comment:
Either put the two global constants at the top of the file, or add them as local constants within the `MakeGasMapForTests()` function.